config/istio/overlays/lock-it-down.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: pilot-network-policy
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      app: pilot
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+    ports:
+    - port: 15004
+  - ports:
+    - port: 15010
+    - port: 15011
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: citadel-network-policy
+  namespace: istio-system
+spec:
+  podSelector:
+    matchLabels:
+      app: citadel
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
```

Comment:
This selector is repeated in all policies, maybe we can move it to ytt function to make it DRY?

Reply:
45136eab0c73c7de3f68ab7c33fcb637e80bc6ea

Reply:
😲